subprojects/java21/java21.gradle

```diff
@@ -1,5 +1,7 @@
 apply from: "$rootDir/gradle/dependencies.gradle"
 apply from: "$rootDir/gradle/java-test.gradle"
+apply from: "$rootDir/gradle/spotless.gradle"
```

Comment:
Can we do the spotless fixes in a separate PR? Good find btw!

In that PR, let's instead add it to `java-library` and `java-test` so that all the other Java subprojects also have it.

Reply:
I will create the other PR.